src/Xamarin.Android.Build.Tasks/Utilities/ManifestDocumentElement.cs

```diff
@@ -31,12 +31,9 @@ public static string ToString (TypeDefinition typeDef, TypeDefinitionCache cache
 
 		public static TypeDefinition ResolveType (string type, ICustomAttributeProvider provider, IAssemblyResolver resolver)
 		{
-			if (type == null)
-				throw new ArgumentException ("Type resolution support requires a non-null Type.", nameof (type));
-			if (provider == null)
-				throw new ArgumentException ("Type resolution support requires an AssemblyDefinition or TypeDefinition.", nameof (provider));
-			if (resolver == null)
-				throw new ArgumentException ("Type resolution support requires a IAssemblyResolver.", nameof (resolver));
+			ArgumentNullException.ThrowIfNull (type);
+			ArgumentNullException.ThrowIfNull (provider);
+			ArgumentNullException.ThrowIfNull (resolver);
```

Comment:
Undo this change `ArgumentNullException.ThrowIfNull()` does not exist in `netstandard2.0`.